openbas-front/src/admin/components/scenarios/scenario/injects/ScenarioInjects.tsx

```diff
@@ -185,66 +185,22 @@ const ScenarioInjects: FunctionComponent<Props> = () => {
     injectContext.onBulkDeleteInjects(injectsToProcess.map((inject: Inject) => inject.inject_id));
   };
 
-  const [openDialog, setOpenDialog] = React.useState<boolean>(false);
-  const handleCloseDialog = (
-    event?: React.SyntheticEvent | Event,
-    reason?: SnackbarCloseReason,
-  ) => {
-    if (reason === 'clickaway') {
-      return;
-    }
-    setOpenDialog(false);
-  };
-  const [detailsLink, setDetailsLink] = React.useState<string>('');
-
-  useEffect(() => {
-    if (openDialog) {
-      setTimeout(() => {
-        handleCloseDialog();
-        setDetailsLink('');
-      }, 6000);
-    }
-  }, [openDialog]);
-
   const massTestInjects = () => {
     bulkTestInjects(injectsToProcess.map((inject: Inject) => inject.inject_id)).then((result: { data: InjectStatus[] }) => {
       if (numberOfSelectedElements === 1) {
-        setDetailsLink(`/admin/scenarios/${scenario.scenario_id}/tests/${result.data[0].status_id}`);
-        setOpenDialog(true);
+        MESSAGING$.notifySuccess(t('Inject test has been sent, you can view test logs details on {itsDedicatedPage}.', {
+          itsDedicatedPage: <Link to={`/admin/scenarios/${scenario.scenario_id}/tests/${result.data[0].status_id}`}>{t('its dedicated page')}</Link>,
+        }));
       } else {
-        setDetailsLink(`/admin/scenarios/${scenario.scenario_id}/tests`);
-        setOpenDialog(true);
+        MESSAGING$.notifySuccess(t('Inject test has been sent, you can view test logs details on {itsDedicatedPage}.', {
+          itsDedicatedPage: <Link to={`/admin/scenarios/${scenario.scenario_id}/tests`}>{t('its dedicated page')}</Link>,
+        }));
```

Comment:
if else if for adding plural but it seems to be the same message ? 
you can do plural within react-intl : https://formatjs.io/docs/react-intl/api/#formatmessage
```
Hello, {name}, you have {itemCount, plural,
    =0 {no items}
    one {# item}
    other {# items}
}.
```

Reply:
To do in a later technical improvement task